docs/_markbind/layouts/default.md

```diff
@@ -0,0 +1,60 @@
+<head-bottom>
+  <link rel="stylesheet" href="{{baseUrl}}/stylesheets/main.css">
+</head-bottom>
+
+<header sticky>
+  <navbar type="dark">
+    <a slot="brand" href="{{baseUrl}}/index.html" title="Home" class="navbar-brand">AB-3</a>
+    <li><a href="{{baseUrl}}/UserGuide.html" class="nav-link">User Guide</a></li>
+    <li><a href="{{baseUrl}}/DeveloperGuide.html" class="nav-link">Developer Guide</a></li>
+    <li><a href="{{baseUrl}}/AboutUs.html" class="nav-link">About Us</a></li>
+    <li><a href="https://github.com/se-edu/addressbook-level3" target="_blank" class="nav-link"><md>:fab-github:</md></a>
+    </li>
+    <li slot="right">
+      <form class="navbar-form">
+        <searchbar :data="searchData" placeholder="Search" :on-hit="searchCallback" menu-align-right></searchbar>
+      </form>
+    </li>
+  </navbar>
+</header>
+
+<div id="flex-body">
+  <nav id="site-nav">
+    <div class="site-nav-top">
+      <div class="fw-bold mb-2" style="font-size: 1.25rem;"></div>
+    </div>
+    <div class="nav-component slim-scroll">
+      <site-nav>
+* [Home]({{ baseUrl }}/index.html)
+* [User Guide]({{ baseUrl }}/UserGuide.html)
```

Comment:
Also give the next level of UG headings, to be on par with how the DG is shown in the site nav?

Reply:
Do you mean adding the titles of the user guide sections below just like the DG? For DG it's added because those are separate pages that are linked from the main DG page. Our UG currently only has one page. Another thing to note is that when printing the UG/DG, only the content on that page will be printed, so those explicitly listed pages wont be included in the DG PDF (they are still available as links from DG, same as the original docs)

Reply:
>Do you mean adding the titles of the user guide sections below just like the DG? For DG it's added because those are separate pages that are linked from the main DG page. Our UG currently only has one page. 

Yes, the main sections of the UG. Yup, they are in the same page but adding them in the sitenav makes the sitenav more balanced, and it is slightly more convenient to jump to a UG section. WDYT?

>Another thing to note is that when printing the UG/DG, only the content on that page will be printed, so those explicitly listed pages wont be included in the DG PDF (they are still available as links from DG, same as the original docs)

Yes, that's the current behavior too, and is fine.